acs-integration-tests/src/test/java/com/ge/predix/test/utils/v2/UaaTestUtil.java

```diff
@@ -80,17 +104,47 @@ public OAuth2RestTemplate createZoneClientAndGetTemplate(final String zoneName,
         return this.acsRestTemplateFactory.getOAuth2RestTemplateForClient(this.tokenUrl, clientId, clientSecret);
     }
 
+    public OAuth2RestTemplate createReadOnlyPolicyScopeClient(final String zoneName) {
+
+        String clientId = "admin-" + zoneName + "-readonly";
```

Comment:
This code is largely duplicated in later methods of this class so consider moving it to a common `private` helper method. You can even abstract out the `create*ScopeClient(...)` calls by implementing a helper `private` inner class that's annotated with `@FunctionalInterface` (see: https://blog.adilakhter.com/2014/01/27/java-8-building-a-trifunction-functional-interface/)